paypro-gateways-woocommerce/includes/paypro/wc/gateway.php

```diff
@@ -1,18 +1,24 @@
 <?php if(!defined('ABSPATH')) exit; // Exit if accessed directly
 
-abstract class PayPro_WC_Gateway_Abstract extends WC_Payment_Gateway
+class PayPro_WC_Gateway extends WC_Payment_Gateway
 {
-    protected $default_title;
+	/**
+	 * panther restapi responses
+	 */
+	protected const PAYPROAPIRES_NOTSUBSCRIPTED = "Not subscribed to money transfer service";
 
-    protected $default_description;
+    protected $default_title;
 
     protected $default_logo;
 
     protected $issuer;
+    protected $gateway_title = '';
+    protected $gateway_description = '';
```

Comment:
Shouldn't it be `__('', 'paypro-gateways-woocommerce')`? or it doesn't matter

Reply:
in my eyes it doesnt matter since its an empty string and it would just translate to an empty string and it is not allowed to use those expressions there. (i dont know if there is a way to have them als default but if it is requested i can add it to the constructer and set them `$this->gateway_title = __('', 'paypro-gateways-woocommerce')` (same for description)

![image](https://user-images.githubusercontent.com/55430516/142646027-5a823c03-143c-4ba1-ab62-2cd6b798f065.png)

also a critical error on wordpress
![image](https://user-images.githubusercontent.com/55430516/142641084-999ff463-fa15-4ecc-8eac-bd2ad779fabb.png)